draft-ietf-httpbis-digest-headers.md

```diff
@@ -403,18 +408,21 @@ on messages associated with the request URI and representation metadata, using
 the `Content-Digest` field.
 
 `Want-Representation-Digest` and `Want-Content-Digest` are Structured Fields
-List (see {{Section 3.2 of STRUCTURED-FIELDS}}):
+List (see {{Section 3.2 of STRUCTURED-FIELDS}}) where:
+
+* members cannot have parameters;
```

Comment:
This case is a bit different, what is good justification to prohibit parameters on want-* fields?

Reply:
Ok.